internal/indexer/implementation.go

```diff
@@ -348,7 +376,36 @@ func (i *Indexer) extractInterfacesAndConcreteTypes(pkgs []*packages.Package) (i
 		}
 	}
 
-	return interfaces, concreteTypes
+	batch := func(pkgBatch []string) error {
+		pkgs, err := i.loadPackage(true, pkgBatch...)
```

Comment:
Is `loadPackage` ever called with `deduplicate = false` in non-test code? It looks like the only place that happens is when `loadPackages` is called with `deduplicate = false` [in a test](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/lsif-go%24+loadPackages%28&patternType=literal&case=yes). Would it make sense to remove the argument and test only the `deduplicate = true` case?

Reply:
Hmm, this does seem unused. Looks like this originated in https://github.com/sourcegraph/lsif-go/pull/138/files?w=1#diff-87c2475baa87005bd3a3001ec33f201feea23caddbbf5a2f249238a2bd32026dR122 with the comment "deduplicate should be true in all cases except TestIndexer_shouldVisitPackage." @slimsag might be able to shed light on this one 💡

Reply:
I don't recall, I could dig in if needed but I'd say go with your gut/instinct here!